### PR TITLE
[ADD] hr_employee_substitute.rst

### PR DESCRIPTION
**Pull request by Drsmartinkus**
Documenation for hr_employee_substitute module.

### CODE REVIEW

**Comment by Drsmartinkus** [comment]
seems to be images are slightly too big will crop them

File: hr.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+:banner: banners/project.jpg
```
Comment (oerp-odoo) [inline hr.rst]: why image name is project? its supposed to be about HR. and usually its png?

File: hr/customizations/hr_employee_substitute.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+=============
+
+To start using this new functionality, module :code:`hr_employee_substitute`
+must be installed.
+
+By default datetime type is used for start/end of substitute, but it is
+possible to simplify it and use date type, by checking Date Type for
+Substitutes. This option can be enabled in Employees Settings
+(:menuselection:`Employees --> Configuration --> Settings`), Under Work
+Organization section by changing value of parameter 'Date Type for Substitutes':
```
Comment (oerp-odoo) [inline hr/customizations/hr_employee_substitute.rst]:
```suggestion
Organization section by checking 'Date Type for Substitutes' option:
```

File: hr/customizations/hr_employee_substitute.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+.. image:: media/hr_employee_substitute_settings.png
+    :align: center
+
+Functionality
+=============
+
+This feature allows to specify substitute employees for other employee and
+select specific date range, when substitution will be valid.
+
+In order to view and create employee substitutes navigate to
```
Comment (oerp-odoo) [inline hr/customizations/hr_employee_substitute.rst]:
```suggestion
In order to view and create employee substitutes, navigate to
```

File: hr/customizations/hr_employee_substitute.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+(:menuselection:`Employees --> Substitutes`)
+
+Here you can see all substitutes in a tree view:
+
+.. image:: media/hr_employee_substitutes_tree_view.png
+    :align: center
+
+In order to create substitutes choose employee, substitute and start/end date
```
Comment (oerp-odoo) [inline hr/customizations/hr_employee_substitute.rst]:
```suggestion
In order to create substitutes, choose employee, substitute and start/end date
```

**Review by oerp-odoo — COMMENTED** [review]
You also do not mention that there is scheduler. By default it runs every hour to check if validity of substitutes need to be updated (some can become valid at that time, some invalid)

File: hr.rst (hunk 1 above)
Comment (Drsmartinkus) [reply hr.rst]: All banners seems to be in jpg, if you look at the image you will see that it fits the purpose, however its true i took an existing banner, will look for a new one.

File: locale/lt/LC_MESSAGES/hr.po [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+#: ../../hr/customizations/hr_employee_substitute.rst:14
+msgid ""
+"By default datetime type is used for start/end of substitute, but it is "
+"possible to simplify it and use date type, by checking Date Type for "
+"Substitutes. This option can be enabled in Employees Settings "
+"(:menuselection:`Employees --> Configuration --> Settings`), Under Work "
+"Organization section checking 'Date Type for Substitutes' option:"
+msgstr ""
+"Pagal standartą datos laiko tipas naudojamas datos su laiku, tačiau ji "
```
Comment (oerp-odoo) [inline locale/lt/LC_MESSAGES/hr.po:60]:
```suggestion
"Pagal nutylėjimą datos laiko tipas naudojamas datos su laiku, tačiau jį "
```

File: locale/lt/LC_MESSAGES/hr.po [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+" įjungiant Datos tipas pavaduotojams nustatymą"
+
+#: ../../hr/customizations/hr_employee_substitute.rst:23
+msgid ""
+"Moreover there is a scheduler which by default runs every hour to check "
+"if validity of substitutes need to be updated. Schedulers can be "
+"enabled/disabled on (:menuselection:`Settings --> Technical --> Scheduled"
+" Actions`)"
+msgstr ""
+"Be to, yra tvarkaraštis, kuris pagal standartinius nustatymus kas valandą patikriną"
```
Comment (oerp-odoo) [inline locale/lt/LC_MESSAGES/hr.po]:
```suggestion
"Be to, yra tvarkaraštis, kuris pagal nutylėjimą kas valandą patikriną"
```

File: locale/lt/LC_MESSAGES/hr.po [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+" ar darbuotojo pavaduotojas vis dar galioja. Suplanuotas veiksmas gali būti"
+" įjungtas/išjungtas (:menuselection:`Nustatymai --> Techninis --> Suplanuoti"
+" veiksmai`)"
+
+#: ../../hr/customizations/hr_employee_substitute.rst:28
+msgid "By default substitutes scheduler is enabled once module is installed"
+msgstr "Pagal standartinius nustatymus pavaduotojų suplanuotas veiksmas yra įjungtas įdiegus modulį"
```
Comment (oerp-odoo) [inline locale/lt/LC_MESSAGES/hr.po]:
```suggestion
msgstr "Pagal nutylėjimą pavaduotojų suplanuotas veiksmas yra įjungtas įdiegus modulį"
```

**Review by oerp-odoo — APPROVED** [review]

File: locale/lt/LC_MESSAGES/hr.po [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+#: ../../hr/customizations/hr_employee_substitute.rst:14
+msgid ""
+"By default datetime type is used for start/end of substitute, but it is "
+"possible to simplify it and use date type, by checking Date Type for "
+"Substitutes. This option can be enabled in Employees Settings "
+"(:menuselection:`Employees --> Configuration --> Settings`), Under Work "
+"Organization section checking 'Date Type for Substitutes' option:"
+msgstr ""
+"Pagal standartą datos laiko tipas naudojamas datos su laiku, tačiau ji "
+"galima pakeisti į datos tipą be laiko. Tai galima nustatytį darbuotojų "
```
Comment (SButko) [inline locale/lt/LC_MESSAGES/hr.po:61]:
```suggestion
"galima pakeisti į datos tipą be laiko. Tai galima nustatyti darbuotojų "
```

File: locale/lt/LC_MESSAGES/hr.po [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+"nustatymuose (:menuselection:`Darbuotojai --> Nustatymai --> Nustatymai`)"
+" įjungiant Datos tipas pavaduotojams nustatymą"
```
Comment (SButko) [inline locale/lt/LC_MESSAGES/hr.po:63]:
```suggestion
" įjungiant Datos tipas pavaduotojams nustatymą:"
```

File: locale/lt/LC_MESSAGES/hr.po [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+
+#: ../../hr/customizations/hr_employee_substitute.rst:23
+msgid ""
+"Moreover there is a scheduler which by default runs every hour to check "
+"if validity of substitutes need to be updated. Schedulers can be "
+"enabled/disabled on (:menuselection:`Settings --> Technical --> Scheduled"
+" Actions`)"
+msgstr ""
+"Be to, yra tvarkaraštis, kuris pagal pagal nutylėjimą kas valandą patikriną"
```
Comment (SButko) [inline locale/lt/LC_MESSAGES/hr.po:72]: Should be:
"Be to, yra suplanuotas veiksmas, kuris pagal pagal nutylėjimą kas valandą patikrina"

File: locale/lt/LC_MESSAGES/hr.po [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+"Norėdami sukurti pavaduotojus, pasirinkite darbuotoją, pavaduotoją, "
+"pradžios ir pabaigos datas ir spustelėkite išsaugojimo mygtuką:"
+
+#: ../../hr/customizations/hr_employee_substitute.rst:50
+msgid ""
+"Employee substitute become's valid automatically once it reaches today's "
+"date:"
+msgstr ""
+"Darbuotojo pavaduotojas automatiškai tampa galiojančiu, kai tik "
+"pasiekiama šiandienos data"
```
Comment (SButko) [inline locale/lt/LC_MESSAGES/hr.po:119]: "pasiekiama šiandienos data:"

File: locale/lt/LC_MESSAGES/hr.po [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+
+#: ../../hr/customizations/hr_employee_substitute.rst:55
+msgid ""
+"Additionally substitutes can be viewed/created directly from Employees "
+"app."
+msgstr ""
+"Papildomai pavaduotojus galima peržiūrėti / sukurti tiesiogiai iš "
+"Darbuotojų applikacijos"
```
Comment (SButko) [inline locale/lt/LC_MESSAGES/hr.po:127]: "Darbuotojų applikacijos."

File: locale/lt/LC_MESSAGES/hr.po [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+msgid ""
+"In order to view and create employee substitutes, navigate to "
+"(:menuselection:`Employees --> Substitutes`)"
+msgstr ""
+"Norėdami peržiūrėti ir sukurti darbuotojų pavaduotojus, eikite į "
+"(:menuselection:`Darbuotojai --> Pavaduotojai`)"
+
+#: ../../hr/customizations/hr_employee_substitute.rst:39
+msgid "Here you can see all substitutes in a tree view:"
+msgstr "Čia galite pamatyti visų pavaduotojų sąrašą"
```
Comment (SButko) [inline locale/lt/LC_MESSAGES/hr.po:103]: "Čia galite pamatyti visų pavaduotojų sąrašą:"

**Pull request merged**